vertical-pod-autoscaler/pkg/admission-controller/resource/pod/recommendation/recommendation_provider.go

```diff
@@ -70,6 +70,22 @@ func GetContainersResources(pod *core.Pod, vpaResourcePolicy *vpa_types.PodResou
 				}
 			}
 		}
+		// Ensure container requests always stay below container limits
```

Comment:
Can you take a look at this: https://github.com/kubernetes/autoscaler/pull/2005/commits/8bc98b75b1ee2c1f23a5221207916e332eeab75a

This is a PR that removed recommendation to limit capping when we started doing proportional limit. The capping is applied to all recommendation parts (Target, Upper and Lower limit). Could you reuse that solution? (sorry for only providing it now)

Reply:
@bskiba updated. Does that look okay now?